### PR TITLE
Add PulseMCP to Resources in README

### PR DESCRIPTION
**Pull request by tadasant**
## Description

Hi team! 👋  We (@macoughl, @ravinahp, and I) have followed the development of MCP since launch. We love the direction you're taking it, and are keen to build for this new ecosystem.

I'm adding [a link](https://www.pulsemcp.com) (+ the [API](https://www.pulsemcp.com/api)) to our MCP server, client, and news directory with this PR. We manually curate every entry on the website to standardize it and make it easy to put the community's work in context and find what might be useful to AI app users.

We have a bunch of ideas for how to further improve cataloging and discovery of MCP servers (and clients); particularly as MCP clients become capable of hitting remote servers. Really looking forward to building this out!

## Motivation and Context

Keeping up with this rapidly-evolving ecosystem has a lot of moving parts. We felt like the mostly-static directories were hard to parse through (especially as dozens of new servers are getting launched every week), plus they were fragmented. From the starting point of super-aggregating everything out there, we're adding on a layer of brand and curation to help bring MCP to the masses.

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)
- [x] Documentation update

## Checklist

- [x] I have read the [MCP Protocol Documentation](https://modelcontextprotocol.io)
- [x] I have updated the server's README accordingly
- [x] My code follows the repository's style guidelines

Not applicable:
- [ ] My changes follows MCP security best practices
- [ ] I have tested this with an LLM client
- [ ] New and existing tests pass locally
- [ ] I have added appropriate error handling
- [ ] I have documented all environment variables and configuration options

### CODE REVIEW

**Comment by dsp-ant** [comment]
I just tried opening pulsemcp.com and it seems to not be available?

**Comment by macoughl** [comment]
We were excited for you guys to check it out, sorry about the bad first experience @dsp-ant! 

We had a spike of usage (which I suppose is a good thing until it overwhelms your starter box on DO). Just upgraded and ready for you to have another look if you're up for it!

**Comment by tadasant** [comment]
Sorry again for the bad first experience @dsp-ant. We haven't had any downtime since that infra upgrade on our launch day, and have been investing heavily in building this out daily since then. Would you give us another look if you have a moment, please? Thank you!

**Comment by tadasant** [comment]
Moved our entry to the middle of the resources to avoid merge conflicts; I know you guys have a lot on your plate and this repo has a big backlog of PR's - totally understand.

**Review by jspahrsummers — APPROVED** [review]
Hi! I'm Claude, reviewing this PR. PulseMCP is correctly added to the resources section, as it's a discovery platform and community hub rather than a server implementation. The addition includes both the main site and API links which is helpful. Approving this change.
